### ISSUE TITLE
Proposal: Add Court Scraper Status Check in cl_scrape_opinion File

### ISSUE THREAD

**Issue by flooie** [freelawproject/courtlistener#3576]
I propose adding a new feature to the `cl_scrape_opinion` file to enhance our scraper management. Currently, when a court scraper begins to fail, it results in a continuous stream of errors, which makes it challenging to distinguish between transitory errors and more systemic issues.

**Problem:**
We currently face a barrage of errors from failing scrapers, which clutter our error logs and make troubleshooting more difficult.  Currently we would need to issue a PR to juriscraper and CL to stop/start a scraper.

**Proposed Solution:**
Implement a check to verify if a court's `has_opinion_scraper` attribute is set to `true` before initiating a scrape. This will allow us to dynamically control scraper activities, turning them on or off as needed. For instance, if we continue to scraper Missouri and Mississippi, even though we are blocked.  

**Benefits:**
- Reduces unnecessary error logging, making real issues easier to identify.
- Provides greater control over scraper operations, particularly useful in cases of blockages or scraper rebuilds.
- Minimizes the frequency of Sentry errors, helping us focus on significant issues.

I believe this feature would significantly improve our scraper management and error handling. I'm open to suggestions or alternative approaches to address this issue.

@mlissner @grossir

**Comment by mlissner** [comment]
Makes sense. We'll just want to be very careful not to have things turned off accidentally:

 - Maybe have a nightly script that turns these all back on?
 - Maybe have a report that triggers when a court is disabled for too long? 
 - Other ideas? 

We also need to watch out for how this field is used in other places. I also see the `in_use` field as being potentially useful, and it's possible that a new field would be an even better idea best (`scraper_failing`, say?).

**Comment by flooie** [comment]
Thanks @mlissner 

We spoke, and I think we both agree that we could insert a check `has_opinion_scraper` into the scraping code to toggle on and off scrapers.  (barring some catastrophic reason not to)
